src/components/TweetBox/TweetBox.jsx

```diff
@@ -3,14 +3,34 @@ import TweetInput from "./TweetInput"
 import "./TweetBox.css"
 
 export default function TweetBox(props) {
+  const handleOnKeyUp = () => {
+    props.setIsDisabledSubmitButton(props.tweetText.length === 0 || props.tweetText.length > 140 ? true : false)
+  }
+  const handleOnTweetTextChange = (changeEvent) => {
+    props.setTweetText(changeEvent.target.value)
+  }
+  let newTweet
```

Comment:
Why does `newTweet` need to be declared out here? Can it be declared on line 14?